aexpect/remote.py

```diff
@@ -19,6 +19,27 @@
 
 """
 Functions and classes used for logging into guests and transferring files.
+
+Example usage:
+
+::
+
+    import aexpect
+    import aexpect.remote
+
+    b = aexpect.ShellSession("sh")
+    b.cmd("echo asdf > /tmp/bbb")
+
+    a = aexpect.remote.remote_login("ssh", "127.0.0.1", 5000, "root", "123456", "[\\#\\$]")
+    a.cmd("rm /tmp/aaa -Rf")
+
+    aexpect.remote.scp_to_reusing_session(a, "/tmp/bbb", "/tmp/aaa")
```

Comment:
this method does not exists, only `scp_to_session`...

Reply:
Hmmm I copy-pasted the example you provided previously wondering if it could be added to the docstring, I guess I took it too literally. Fixed now.